Tests/Marketplace/marketplace_services.py

```diff
@@ -628,7 +628,7 @@ def _load_pack_dependencies(self, index_folder_path, first_level_dependencies, a
         return dependencies_data_result
 
     def _create_changelog_entry(self, release_notes, version_display_name, build_number, pack_was_modified=False,
-                                new_version=True, initial_release=False):
+                                new_version=True, initial_release=False, initial_entry_time=None):
```

Comment:
I think we should come up with a `update_changelog_entry` function that can get all changed parts of a changelog entry and change them accordingly, we want to keep the build number the same as well.

Reply:
Done and tested in this build https://app.circleci.com/pipelines/github/demisto/content/98938/workflows/ec611eac-001e-43db-a773-0f0a500060e2/jobs/394329